.sync.yml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
What's the purpose of this file?

Reply:
Part of the conversion we moved the module under [PDK](https://puppet.com/docs/pdk/1.x/pdk.html) and have been making use of `pdk update`. 

This file determines what additional gems, gitignore paths etc. are required on top of the ones set out in the [pdk-templates](https://github.com/puppetlabs/pdk-templates).